documentation/advanced/tutorial-switch-npm-pnpm.asciidoc

```diff
@@ -1,40 +1,58 @@
 ---
-title: Differences between NPM and PNPM
+title: Switch between NPM and PNPM
 order: 16
 layout: page
 ---
 
 ifdef::env-github[:outfilesuffix: .asciidoc]
 
-= Differences Between pnpm and npm
+= Switch between NPM and PNPM
 
-Starting from Vaadin 14.2 and 15 _pnpm_ (AKA _performant npm_) tool is used as the default frontend package manager instead of _npm_.
-https://pnpm.js.org/[pnpm] uses a shared repository for all projects in a system which allows to reduce
-packages download time. On the contrast, by default npm will download all dependencies again for every project.
+Vaadin 15 uses `npm` as the default package manager. It's also possible to switch to https://pnpm.js.org/[pnpm], the benefit of `pnpm` is that it uses a shared repository, which allows reducing the build time.
```

Comment:
This should be fixed and mention that "npm is the default package manager. Starting from 14.2 and 15, it is possible to switch to pnpm..."